docs/source/aryn_cloud/aryn_partitioning_service.rst

```diff
@@ -1,16 +1,18 @@
 Aryn Partitioning Service
 =========================
 
-You can use the Aryn Partitioning Service to segment PDFs into labeled bounding boxes identifying titles, tables, table rows and columns, images, and regular text. Bounding boxes are returned as JSON with their associated text for easy use. It leverages `Aryn's purpose-built AI model for document segmentation and labelling <https://huggingface.co/Aryn/deformable-detr>`_ that was trained using DocLayNet – an open source, human-annotated document layout segmentation dataset containing tens of thousands of pages from a broad variety of document sources.
+You can use the Aryn Partitioning Service to segment PDFs and Microsoft Office (.doc, .docx, .ppt, .pptx) documents and to automatically extract table data.  The service labels titles, tables, table rows and columns, images, and regular text. Bounding boxes are returned as JSON with their associated text for easy use. It leverages `Aryn's purpose-built AI model for document segmentation and labelling <https://huggingface.co/Aryn/deformable-detr>`_ that was trained using DocLayNet – an open source, human-annotated document layout segmentation dataset containing tens of thousands of pages from a broad variety of document sources.
```

Comment:
Office isn't used anymore. I'd replace this with "Microsoft Word (.docx and .dox), and Microsoft PowerPoint (.pptx and .ppt)"

Reply:
Done.